assets/xml/objects/object_sekihig.xml

```diff
@@ -1,16 +1,16 @@
 ﻿<Root>
     <File Name="object_sekihig" Segment="6">
-        <DList Name="object_sekihig_DL_000340" Offset="0x340" />
-        <DList Name="object_sekihig_DL_0004A8" Offset="0x4A8" />
-        <DList Name="object_sekihig_DL_0004B8" Offset="0x4B8" />
-        <DList Name="object_sekihig_DL_0005C8" Offset="0x5C8" />
+        <DList Name="gSunSongGoronSymbolBaseDL" Offset="0x340" />
+        <DList Name="gSunSongGoronSymbolDL" Offset="0x4A8" />
+        <DList Name="gSekihigGraveBaseDL" Offset="0x4B8" />
+        <DList Name="gGraveGoronSymbolDL" Offset="0x5C8" />
         <Texture Name="object_sekihig_TLUT_0005E0" OutName="tlut_0005E0" Format="rgba16" Width="4" Height="4" Offset="0x5E0" />
```

Comment:
```suggestion
        <Texture Name="gSekihiTLUT" OutName="sekihi_tlut" Format="rgba16" Width="4" Height="4" Offset="0x5E0" />
```
Since it is just the one tlut, can just name it as such. This applies to the other TLUTs in the other objects as well.